docs/application/toc_all.md

```diff
@@ -1174,6 +1175,7 @@
 #### Configure Your App
 ##### [Overview](/application/tizen-studio/native-tools/configuring.md)
 ##### [Manifest Text Editor](/application/tizen-studio/native-tools/manifest-text-editor.md)
+##### [Manifest Element](/application/tizen-studio/native-tools/manifest-element.md)
```

Comment:
```suggestion
##### [Manifest Element](/application/tizen-studio/native-tools/metadata-element.md)
```

Reply:
fixed title and filename. manifest(x) => metadata (o)